test/models/imports/episode_rss_import_test.rb

```diff
@@ -0,0 +1,166 @@
+require "test_helper"
+
+describe EpisodeRssImport do
```

Comment:
This test and `podcast_rss_import_test.rb` are unchanged - just copied as-is out of the previous base class tests.

(Though I did also cleanup the `test/fixtures` that weren't being used - appeared to be some code copied out of CMS that wasn't necessary here).